reverb/pip_package/setup.py

```diff
@@ -125,6 +135,9 @@ def run_setup(self):
         packages=find_packages(),
         headers=list(find_files('*.proto', 'reverb')),
         include_package_data=True,
+        package_data={
+            'reverb': matches,
```

Comment:
why did this work for .so files in linux but not on macos?  can we use the same code paths for both?

Reply:
ditto